app/javascript/components/components/Header.js

```diff
@@ -33,22 +34,25 @@ export default class Example extends React.Component {
     } = this.props;
     return (
       <nav>
-        <Navbar color="light" light expand="md">
-          <NavbarBrand href="/">Apartment App</NavbarBrand>
+        <Navbar color="dark" dark expand="md">
+          <NavbarBrand href="/">
+            <img src={keyfinder} alt="Key Finder Logo" />
+            Apartment App
+          </NavbarBrand>
           <NavbarToggler onClick={this.toggle} />
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className="ml-auto" navbar>
-              <NavItem>
-                <NavLink href="/apartmentindex">Listings</NavLink>
-              </NavItem>
               <NavItem>
                 {!logged_in && (
                   <NavLink href={sign_in_route}>Sign In/Register</NavLink>
                 )}
                 {logged_in && (
-                  <NavLink href={sign_out_route}>My Listings</NavLink> // placeholder href
+                  <NavLink href="/apartmentprotectedindex">My Listings</NavLink>
```

Comment:
Any internal React navigation should be done with the react-router NavLink and a `to` attribute.

Reply:
Thanks for the feedback!  I'm using the reactstrap version of NavLink and it doesn't seem to like `to`.  When I import the react-router-dom version of NavLink, it seems to break when using `href={sign_in_route}` and `to="/apartmentprotectedindex"`.  Also the styling gets all messed up because of reactstrap.  Is there a way around this?